client/src/assets/player/peertube-player-manager.ts

```diff
@@ -242,8 +246,11 @@ export class PeertubePlayerManager {
         videoCaptions: commonOptions.videoCaptions,
         stopTime: commonOptions.stopTime,
         isLive: commonOptions.isLive,
-        videoUUID: commonOptions.videoUUID
-      }
+        videoUUID: commonOptions.videoUUID,
+      },
+      chromecast: {
+        
+      },
```

Comment:
Behövs detta?

Reply:
In videojs you load plugins by specifing it in the plugins object like this, or by doing `player.chromecast()`. You can change to the latter if it makes more sense to you.